translations/DNC.js

```diff
@@ -41,6 +41,10 @@ function initialize()
     // Set the schema type for the export
     hoot.Settings.set({"osm.map.writer.schema":"DNC"});
     hoot.Settings.set({"ogr.esri.fdname":""});
+
+    // We need this to avoid errors with int & LongLong
+    // It really needs to be set on the commandline not here
+    hoot.Settings.set({"ogr.strict.checking":"off"});
```

Comment:
So this is necessary but not sufficient to avoid the min int errors which is why we add the option again in the command lines above?

Reply:
Yes.
Putting it in the Javascript reminds me that it is needed. It doesn't hurt to put it here as well but it doesn't fix the problem.
Since the javascript is initialised AFTER the OgrWriter, setting the the config variable in the java  

E.g. Adding debug code to the C++ and setting the config variable only in the javascript
The initialization:
```
01:03:19.113 INFO  ...hoot/core/io/DataConverter.cpp( 184) Converting ../A1708280_all.osm to export_A1708280_all.shp...
01:03:19.120 INFO  ...hoot/core/io/DataConverter.cpp( 309) _transToOgrMT
01:03:28.033 INFO  ...hoot/core/io/DataConverter.cpp( 303) Done Reading
01:03:28.061 INFO  ...hoot/core/io/DataConverter.cpp( 324) Element Cache Filled
01:03:28.072 INFO  ...hoot/core/io/DataConverter.cpp( 340) Translation Thread Started
01:03:28.074 INFO  ...hoot/core/io/DataConverter.cpp( 351) OGR Writer Thread Started
01:03:28.075 INFO  ...hoot/core/io/DataConverter.cpp( 354) Waiting for writer to finish...
01:03:28.077 INFO  ...cpp/hoot/core/io/OgrWriter.cpp( 475) strictStr: on
01:03:28.079 INFO  ...chema/JavaScriptTranslator.cpp( 239) Loading translation script: /home/dg/hoot2/translations/DNC.js
01:03:28.172 INFO  ...cpp/hoot/core/io/OgrWriter.cpp( 475) strictStr: off
01:03:28.172 INFO  ...chema/JavaScriptTranslator.cpp( 239) Loading translation script: /home/dg/hoot2/translations/DNC.js
```
The values in the feature::
```
TableName 0: INDUSTP  FCode: AL240  Geom: Point
Out Attrs:ACC: :1:
Out Attrs:F_CODE: :AL240:
Out Attrs:HGT: :-2147483648:
Out Attrs:LOC: :-32768:
Out Attrs:NAM: :N/A:
Out Attrs:OSM_TAGS: :<OSM>{"source":"dnc:industp_cul"}</OSM>:
Out Attrs:OSM_UUID: :c3e78eb6-4c71-45a6-8b37-6ced75a9e86e:
Out Attrs:PRO: :-32768:
Out Attrs:SSC: :-32768:
Out Attrs:TUC: :-32768:
Out Attrs:USE: :-32768:
Out Attrs:ZV2: :-2147483648:
```

The exception that Hoot throws:
```
terminateHandler caught unhandled exception. what(): Value is below min. value: -2147483648 min: -2147483647 (HGT)
stack trace:
  hoot::SignalCatcher::terminateHandler() +0x5e
  /usr/lib64/libstdc++.so.6 : ()+0x5e746
  /usr/lib64/libstdc++.so.6 : ()+0x5e773
  /usr/lib64/libstdc++.so.6 : ()+0x5e993
  hoot::FieldDefinition::_reportError(QString const&, QString const&, hoot::StrictChecking) const +0x134
<...>
```

Setting the config variable on the command line:
Initialization:
```
01:00:17.384 INFO  ...hoot/core/io/DataConverter.cpp( 184) Converting ../A1708280_all.osm to export_A1708280_all.shp...
01:00:17.390 INFO  ...hoot/core/io/DataConverter.cpp( 309) _transToOgrMT
01:00:25.773 INFO  ...hoot/core/io/DataConverter.cpp( 303) Done Reading
01:00:25.801 INFO  ...hoot/core/io/DataConverter.cpp( 324) Element Cache Filled
01:00:25.811 INFO  ...hoot/core/io/DataConverter.cpp( 340) Translation Thread Started
01:00:25.813 INFO  ...hoot/core/io/DataConverter.cpp( 351) OGR Writer Thread Started
01:00:25.814 INFO  ...hoot/core/io/DataConverter.cpp( 354) Waiting for writer to finish...
01:00:25.816 INFO  ...cpp/hoot/core/io/OgrWriter.cpp( 475) strictStr: off
01:00:25.819 INFO  ...chema/JavaScriptTranslator.cpp( 239) Loading translation script: /home/dg/hoot2/translations/DNC.js
01:00:25.940 INFO  ...cpp/hoot/core/io/OgrWriter.cpp( 475) strictStr: off
01:00:25.941 INFO  ...chema/JavaScriptTranslator.cpp( 239) Loading translation script: /home/dg/hoot2/translations/DNC.js
```
No errors!